Source/GUI/MemWatcher/Dialogs/DlgAddWatchEntry.cpp

```diff
@@ -424,3 +437,23 @@ MemWatchEntry* DlgAddWatchEntry::stealEntry()
   m_entry = nullptr;
   return entry;
 }
+
+void DlgAddWatchEntry::onPointerOffsetContextMenuRequested(const QPoint& pos)
+{
+  QLabel* const lbl = static_cast<QLabel*>(sender());
+
+  QMenu* contextMenu = new QMenu(this);
+  QAction* copyAddr = new QAction(tr("&Copy Address"), this);
+
+  const QString text{QString::number(lbl->property("addr").toUInt(), 16).toUpper()};
+  connect(copyAddr, &QAction::triggered, this,
+    [text] { QApplication::clipboard()->setText(text); });
+  contextMenu->addAction(copyAddr);
+
+  if (!lbl->property("addr").toUInt())
+    {
+        copyAddr->setEnabled(false);
+      }
```

Comment:
Nit: The indentation has gone bad around these lines (and perhaps in other lines too). The project uses Clang-Format for formatting; there is a `.clang-format` file in the repository.

One thing the build does in the CI is check that the formatting is correct. So, I'm guessing (and hoping!) that the CI would flag this as an error.